### ISSUE TITLE
Alternative loss functions

### ISSUE THREAD

**Issue by JamesZDonline** [JamesZDonline/ArcheoStructures_RV#3]
Basic cross-entropy may not be the best loss function due to the unbalanced training set.

It'd be nice to try out focal loss (or perhaps other loss functions if there's something better).

RasterVision has the ability to use external loss functions (especially off of TorchHub) so this shouldn't be hard to accomplish but would be good to at least attempt.
